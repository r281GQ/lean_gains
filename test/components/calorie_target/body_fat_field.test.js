import React from 'react';
import { shallow } from 'enzyme';
import BodyFatField from './../../../src/components/calorie_target/body_fat_field';

describe('BodyFatField test', () => {
  it('should render', () => {
    const component = shallow(
      <BodyFatField bmrCalculationMethod="katch-mcardle" />
    );
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
