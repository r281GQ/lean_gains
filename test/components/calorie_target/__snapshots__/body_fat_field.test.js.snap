// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BodyFatField test should render 1`] = `
<Field
  component={[Function]}
  floatingLabelText="You can manually enter your bodyfat %"
  fullWidth={true}
  max={50}
  min={0}
  name="bodyFat"
  step={0.1}
  type="number"
/>
`;
